docs/RELEASE_NOTES-v0.4.md

```diff
@@ -10,18 +11,42 @@ versions of `fedimintd` requires stopping all peers at the exact same session co
 (mint's internal consensus height), and switching to new v0.4.x release at the same time,
 before starting them again.
 
+All guardians should be available to coordinate upgrading using an out-of-band communicaitons
+channel, e.g. a group chat. Once all guardians are available, get the latest session count so
+a session in the future can be chosen to coordinate a shutdown.
 
-To schedule stopping your `fedimintd` at specific consensus height you can
-use `fedimin-cli` command:
+Get the current session count using `fedimint-cli`:
 
-**Details to be described in the near feature. Check newer version of this document.**
+```
+fedimint-cli session-count
+```
+
+The minimum time required to complete a session is 3 minutes, so choose a future session that
+will give all guardians enough time to schedule a shutdown after the session count (e.g. 15
+minutes implies 5 sessions after the current session count).
+
+To schedule stopping your `fedimintd` after a specific session count, use the
+`fedimint-cli` command:
+
+```
+fedimint-cli --password <password> --our-id <our_id> admin shutdown <session_count>
+```
+
+Alternatively, the current session count and scheduling a shutdown after session count is
```

Comment:
@Kodylow I'm going off of memory from the UI screenshare, so please let me know if this isn't an accurate description.